e2e/e2etests/test_bitcoin_deposit_and_call_revert_with_dust.go

```diff
@@ -38,16 +37,13 @@ func TestBitcoinDepositAndCallRevertWithDust(r *runner.E2ERunner, args []string)
 	// Send BTC to TSS address with a dummy memo
 	// zetacore should revert cctx if call is made on a non-existing address
 	nonExistReceiver := sample.EthAddress()
-	badMemo := append(nonExistReceiver.Bytes(), []byte("gibberish-memo")...)
-	txHash, err := r.SendToTSSFromDeployerWithMemo(amount, utxos, badMemo)
+	anyMemo := append(nonExistReceiver.Bytes(), []byte("gibberish-memo")...)
+	txHash, err := r.SendToTSSFromDeployerWithMemo(amount, utxos, anyMemo)
 	require.NoError(r, err)
 	require.NotEmpty(r, txHash)
 
-	// wait for the cctx to be mined
-	cctx := utils.WaitCctxMinedByInboundHash(r.Ctx, txHash.String(), r.CctxClient, r.Logger, r.CctxTimeout)
-	r.Logger.CCTX(*cctx, "deposit_and_revert")
-	utils.RequireCCTXStatus(r, cctx, crosschaintypes.CctxStatus_Reverted)
-
-	// check the test was effective: the revert outbound amount is less than the dust amount
-	require.Less(r, cctx.GetCurrentOutboundParam().Amount.Uint64(), uint64(constant.BTCWithdrawalDustAmount))
+	// ASSERT
+	// Now we want to make sure the cctx is aborted.
+	cctx := utils.WaitCctxAbortedByInboundHash(r.Ctx, r, txHash.String(), r.CctxClient)
+	require.True(r, cctx.GetCurrentOutboundParam().Amount.Uint64() < constant.BTCWithdrawalDustAmount)
```

Comment:
We might want to add an assertion in the status message to check it abort because of dust amount, not because the amount is not sufficient to pay withdraw fee

Reply:
https://github.com/zeta-chain/node/pull/3149/commits/46ede19a9677cf98d7ce9412e0e4d45eaa9bf118